packages/db/src/migrations/20250915152638_create_granule_duplicates_table_and_index.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  await knex.schema.createTable('granule_duplicates', (table) => {
+    table
+      .increments('cumulus_id')
+      .primary();
+    table
+      .integer('granule_cumulus_id')
+      .references('cumulus_id')
+      .inTable('granules')
+      .notNullable();
+    table
+      .specificType('status', 'char(1)')
+      .comment('Granule active status')
+      .notNullable();
+    table
+      .string('group_id')
+      .comment('Granule duplicate group id')
+      .notNullable();
+  });
+
+  await knex.raw('CREATE INDEX IF NOT EXISTS granule_duplicates_granule_cumulus_id_index ON granule_duplicates(granule_cumulus_id)');
+  await knex.raw('CREATE INDEX IF NOT EXISTS granule_duplicates_group_id_index ON granule_duplicates(group_id)');
```

Comment:
What's the thinking around indexing both of these fields?

Reply:
This is mostly irrelevant as we're now making the FK the primary key (and it will be indexed as a result).